### PR TITLE
[zh] Update reference\glossary (7)

### PR DESCRIPTION
**Pull request by ydFu**
Update about reference/glossary,this chapter is often used, so make adjustments,and ensure all these files were still in sync with English upstream now.

the following
```
content/zh-cn/docs/reference/glossary/cni.md
content/zh-cn/docs/reference/glossary/container-runtime-interface.md
content/zh-cn/docs/reference/glossary/container-runtime.md
content/zh-cn/docs/reference/glossary/container.md
content/zh-cn/docs/reference/glossary/containerd.md
```
Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh-cn/docs/reference/glossary/cni.md [hunk 1]
```diff
@@ -11,31 +11,28 @@ aka:
...
 
-<!--more--> 
+<!--more-->
 
 <!--
-* For information on Kubernetes and CNI, see [**Network plugins**](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni).
+* For information on Kubernetes and CNI, see [**Network Plugins**](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/).
 -->
-* 想了解 Kubernetes 和 CNI 请参考[**网络插件**](/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni)。
+* 有關 Kubernetes 和 CNI 的信息，请参考 [**网络插件**](/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/cni.md:38]:
```suggestion
* 有关 Kubernetes 和 CNI 的信息，请参考[**网络插件**](/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)。
```

File: content/zh-cn/docs/reference/glossary/container.md [hunk 2]
```diff
@@ -1,40 +1,38 @@
 ---
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/container.md:1]: 这个文件改错了。
英文下面有 container.md 和 containerd.md 两个文件。

Comment (ydFu) [reply content/zh-cn/docs/reference/glossary/container.md:1]: sorry. I check it

File: content/zh-cn/docs/reference/glossary/cni.md [hunk 3]
```diff
@@ -1,41 +1,38 @@
 ---
-title: 容器网络接口（CNI）
+title: 容器网络接口（Container network interface；CNI）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/glossary/cni.md:2]:
```suggestion
title: 容器网络接口（Container network interface，CNI）
```
Normally, it is `,` in en or `，` in zh, less `；` found from my experience.

Comment (ydFu) [reply content/zh-cn/docs/reference/glossary/cni.md:2]: All punctuation needs to be converted according to the [guidelines](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%A0%87%E7%82%B9%E7%AC%A6%E5%8F%B7)
These two words(`Container network interface` and `CNI`) are synonyms, so according to the word [baidu](https://baike.baidu.com/item/%E5%88%86%E5%8F%B7?fromModule=lemma_search-box) needs to be parallel(`并列`), so use a semicolon9(`；`).

Comment (windsonsea) [reply content/zh-cn/docs/reference/glossary/cni.md:2]: ok, all right. It is easy to understand both. For a title style, capitalization might be required, such as:

title: 容器网络接口（Container Network Interface；CNI）

**Pull request merged**